wagtail/wagtailimages/utils.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, unicode_literals
+
+
+# Helper functions for migrating the Rendition.filter foreign key to the filter_spec field,
+# and the corresponding reverse migration
+
+def get_fill_filter_spec_migrations(app_name, rendition_model_name):
+
+    def fill_filter_spec_forward(apps, schema_editor):
+        # Populate Rendition.filter_spec with the spec string of the corresponding Filter object
+        Rendition = apps.get_model(app_name, rendition_model_name)
+        db_alias = schema_editor.connection.alias
+        for rendition in Rendition.objects.using(db_alias).select_related('filter'):
```

Comment:
If I remember correctly, the `update()` method from the QuerySet does not work during data migration. But I'm not sure I tried a raw query, it would be better indeed.